src/app/(dashboard)/recipes/[id]/recipeForm.tsx

```diff
@@ -105,18 +109,35 @@ function RecipeFormInner({
           />
           <Link
             href={recipe.url}
-            className="text-muted-foreground pl-1 text-lg"
+            className="pl-1 text-lg text-muted-foreground"
             target="_blank"
           >
             View Original
           </Link>
         </div>
+          <Tabs.Root defaultValue="ingredients">
+            <Tabs.List className="flex-row border-b">
+              <Tabs.Trigger className="m-r-1 p-4 radix-state-active:rounded-t-md radix-state-active:border-x radix-state-active:border-t" value="ingredients">Ingredients</Tabs.Trigger>
+              <Tabs.Trigger className="m-1 p-4 radix-state-active:rounded-t-md radix-state-active:border-x radix-state-active:border-t" value="steps">Steps</Tabs.Trigger>
+            </Tabs.List>
         <div className="flex flex-col gap-2">
-          <IngredientTable grocyBaseUrl={grocyBaseUrl ?? ""} />
+          <Tabs.Content value="ingredients">
+            <IngredientTable grocyBaseUrl={grocyBaseUrl ?? ""} />
+          </Tabs.Content>
+          <Tabs.Content value="steps">
+            <Controller
+              render={({ field }) => (
+                <ReactQuill value={field.value} onChange={field.onChange}/>
+              )}
+              name="method"
+              control={form.control}
+            />
+          </Tabs.Content>
```

Comment:
I love this - please could we default to using the Shadcn pattern though of defining styled components in the `components/ui` directory rather than inline styling the Radix primitives - this will make it much easier to reuse them in future :)